package.json

```diff
@@ -78,13 +78,15 @@
     "feathers-sequelize": "^6.1.0",
     "helmet": "^4.0.0",
     "listr": "^0.14.3",
+    "peer-id": "^0.14.2",
```

Comment:
Shouldn't be this a `devDependency`?

Reply:
Yes, but we will have it in deps when comms integration comes

Reply:
Hmm I don't think so, because you won't need custom handling of PeerId so you can leave the creation up to the pubsub library. But lets leave it here for now.